content/blog/2018-05-02-onboarding-is-work.md

```diff
@@ -0,0 +1,325 @@
+--- 
+slug: onboarding-is-work
+title: 'How much work is onboarding?'
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+  - name: Noam Ross
+    url: http://twitter.com/noamross
+date: 2018-05-02
+preface: "This blog post is the second of a 3-post series about a data-driven overview of rOpenSci onboarding. Read the intro to the series [here](https://ropensci.org/blog/2018/04/26/satrday-ct-serie/) and the first post about data collection [here](https://ropensci.org/blog/2018/04/26/rectangling-onboarding/)."
```

Comment:
fix intro link. should be https://ropensci.org/blog/2018/04/26/A-satrday-ct-series/